backend/src/main/java/ca/bc/gov/hlth/mohums/controller/GroupsController.java

```diff
@@ -15,7 +15,7 @@ public GroupsController(WebClientService webClientService) {
     }
 
     @GetMapping("/groups")
-    public Mono<Object> getGroups() {
+    public ResponseEntity<Object> getGroups() {
```

Comment:
What I'd like to do is have the `WebClientService` only return the Spring MVC style synchronous interfaces. Totally encapsulate all that `Mono` and `Flux` inside the class.

Reply:
Agree, can we just do that in this PR? The only one missing is the `getClients()`, it should be possible to make it the same as the `getClient()`